Library/Formula/scw.rb

```diff
@@ -23,6 +23,10 @@ def install
 
     system "go", "build", "-o", "#{bin}/scw", "-v", "-ldflags", "-X  github.com/scaleway/scaleway-cli/pkg/scwversion.GITCOMMIT=homebrew", "github.com/scaleway/scaleway-cli/cmd/scw/"
 
+ # we remove our .scw-cache.db file, to clean the cache of scw
+ # this file is created and handled by scw
+    rm_f "~/.scw-cache.db"
```

Comment:
We don't allow writing outside of the temporary installation directory during the installation.  And this seems like something that should be handled by the software internally.